simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderReadiumJavaScriptAPI.java

```diff
@@ -237,30 +235,10 @@ private void evaluateWithResult(
         NullCheck.notNull(
           String.format("ReadiumSDK.reader.updateSettings(%s);", vs.toJSON())));
 
-    } catch (final JSONException e) {
-      ReaderReadiumJavaScriptAPI.LOG.error(
-        "error constructing json: {}", e.getMessage(), e);
-    }
-  }
-
-  @Override public void injectFonts()
-  {
-    try {
-      final JSONObject s = new JSONObject();
-      s.put("truetype", "OpenDyslexic3-Regular.ttf");
-
-      final JSONObject o = new JSONObject();
-      o.put("fontFamily", "OpenDyslexic3");
-      o.put("fontWeight", "normal");
-      o.put("fontStyle", "normal");
-      o.put("sources", s);
-
-      final StringBuilder script = new StringBuilder(256);
-      script.append("ReadiumSDK.reader.plugins.injectFonts.registerFontFace(");
-      script.append(o);
-      script.append(");");
-
-      this.evaluate(script.toString());
+      // Update the selected user font through the custom Simplified JS script.
+      this.evaluate(
+        NullCheck.notNull(
+          String.format("simplified.updateBookStyles({ 'font-family': '%1s', color: '%2s'});", fontSelected, color)));
```

Comment:
Now also updating the text color for books that did not previously update. Specifically, this is for the white text and black background theme.